10313.xml

```diff
@@ -0,0 +1,241 @@
+30 = Every 30 seconds
+600 = Every 10 minutes
+1800 = Every 30 minutes
+3600 = Hourly
+7200 = Every 2 hours
+14400 = Every 4 hours
+43200 = Every 12 hours
+86400 = Every Day
+172600 = Every Second Day]]></Description>
+            </Item>
+            <Item ID="6001">
+                <Name>Interval Start Offset</Name>
+                <Operations>R</Operations>
+                <MultipleInstances>Single</MultipleInstances>
+                <Mandatory>Optional</Mandatory>
+                <Type>Integer</Type>
+                <RangeEnumeration>0..86399</RangeEnumeration>
+                <Units>s</Units>
+                <Description><![CDATA[The Interval Start Offset resource is a read only resource representing the number of seconds past midnight for which the first interval is recorded. If this resource is empty, it is assumed that the intervals are midnight aligned. This can be used to adjust interval boundaries. As an example, an Interval Period of 3600 seconds and an Interval Start time of 300 represents hourly interval data, starting at 00:05.]]></Description>
+            </Item>
+            <Item ID="6002">
+                <Name>Interval UTC Offset</Name>
+                <Operations>R</Operations>
+                <MultipleInstances>Single</MultipleInstances>
+                <Mandatory>Optional</Mandatory>
+                <Type>String</Type>
+                <RangeEnumeration></RangeEnumeration>
+                <Units></Units>
+                <Description><![CDATA[The Interval UTC Offset resource is a read only resource representing the time zone offset for this Interval Data instance. If this resource is empty, the application should use the UTC offset provided in the Device [/3/0/14] object instance resource or UTC if not provided. UTC+X [ISO 8601].]]></Description>
+            </Item>
+            <Item ID="6003">
+                <Name>Interval Collection Start Time</Name>
+                <Operations>R</Operations>
+                <MultipleInstances>Single</MultipleInstances>
+                <Mandatory>Mandatory</Mandatory>
+                <Type>Time</Type>
+                <RangeEnumeration></RangeEnumeration>
+                <Units></Units>
+                <Description><![CDATA[The Collection Start Time resource is a read only resource representing the time when the first interval was recorded on the device. Interval times represent the end of the interval, not the beginning. As an example, the first four hourly interval past midnight will have a timestamp of 04:00 (adjusting for UTC offset). ]]></Description>
```

Comment:
Where does the example 04:00 time come from? The "end of interval" part was a bit confusing.

Reply:
This is purely a reference example to clarify that how the timestamp works. For example "end of the interval" means that for something that is recording data every four hours, the timestamp will be at the end of the 4 hour period, not the start. Its an important distinction but if still not clear we can add more